Sources/SWBGenericUnixPlatform/Plugin.swift

```diff
@@ -73,9 +77,9 @@ struct GenericUnixSDKRegistryExtension: SDKRegistryExtension {
 
         let defaultProperties: [String: PropertyListItem]
         switch operatingSystem {
-        case .linux:
+        case .linux, .freebsd:
```

Comment:
Does the OpenBSD linker support the flag or should that be listed here too?

Reply:
No idea, I don't have an OpenBSD toolchain. I just made the "obvious" changes while I was here since at least one person is working on support for that platform.